test/tokbox_test.js

```diff
@@ -130,7 +131,8 @@ describe("TokBox", function() {
 
       tokBox.getSessionTokens(function(error, info) {
         expect(error).eql(null);
-        assert.calledOnce(statsdSpy);
+        assert.calledOnce(statsdTimer);
+        assert.calledOnce(statsdCount);
```

Comment:
Maybe we want to assert the name of the statsd key ?
Maybe not, since it is probably not crucial in terms of specs.